### PR TITLE
fix(js) do not flag `import()` as a function, rather a keyword 

### PR DESCRIPTION
**Pull request by nathnolt**
SEE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import

No issue made for this.

### Changes
Added the import operator function statement to the list of BUILT_IN_VARIABLES
and to the noneOf inside of the FUNCTION_CALL object, (although I'm not 100% sure if this is required)

### Checklist
- [ ] Added markup tests, or they don't apply here because...
- [ ] Updated the changelog at `CHANGES.md`

Does such a small edit need an edit of changes.md?

### CODE REVIEW

**Comment by joshgoebel** [comment]
> this.buildDefinitions(data["palette"] ?? {}),

Yes.

I wonder though if this shouldn't be a built-in global or even a keyword?  variable seems wrong OTTOMH.

**Comment by nathnolt** [comment]
I don't know man, I just changed it for myself and thought, I guess I could make a pull request.

**Comment by nathnolt** [comment]
Also: how do I test out the highlighting, like have it spit out a demo page where I can insert code into, so I can test out if the changes I've made work or not.

**Comment by joshgoebel** [comment]
> so I can test out if the changes I've made work or not.

See the Building and Testing documentation.  Build a browser release. Then fire up `./tools/developer.html`, that's the easiest way.

File: CHANGES.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ## Version 11.7.0
 
 Grammars:
-- Added 3rd party LookML grammar to SUPPORTED_LANGUAGES [Josh Temple][]
+- enh(js) added `import` operator
```
Comment (joshgoebel) [inline CHANGES.md:4]: Please add credit here and a link below (line ~7)...

File: src/languages/lib/ecmascript.js [hunk 2]
```diff
@@ -141,6 +141,7 @@ export const BUILT_IN_VARIABLES = [
   "arguments",
   "this",
   "super",
+  "import",
```
Comment (joshgoebel) [inline src/languages/lib/ecmascript.js:144]: We already consider this a `keyword`, is there a reason you think it's a built-in variable?  Typically this is used for "objects"...

File: src/languages/javascript.js [hunk 3]
```diff
@@ -360,7 +360,8 @@ export default function(hljs) {
       /\b/,
       noneOf([
         ...ECMAScript.BUILT_IN_GLOBALS,
-        "super"
+        "super",
+        "import"
```
Comment (joshgoebel) [inline src/languages/javascript.js:364]: When the functional version of import is used `import()` should we not highlight it as a function?  Thoughts?

**Comment by joshgoebel** [comment]
> I just changed it for myself and thought, I guess I could make a pull request.

What problem were you trying to solve with this patch?

File: CHANGES.md (hunk 1 above)
Comment (nathnolt) [reply CHANGES.md:4]: Like this?

File: src/languages/lib/ecmascript.js (hunk 2 above)
Comment (joshgoebel) [reply src/languages/lib/ecmascript.js:144]: This addition can be removed since we already highlight it as a keyword.

Comment (nathnolt) [reply src/languages/lib/ecmascript.js:144]: Okay, but wouldn't that make it not have that same color when used as a function, aka `import X from 'y.js'` being colored but `const x = import('y.js')` would just become the default function color?

Comment (joshgoebel) [reply src/languages/lib/ecmascript.js:144]: No, that is what `noneOf` resolves... you should be able to test both and see the results.

**Comment by joshgoebel** [comment]
Need to close this up soon if you'd like it in 11.7... that may be the last release before 12.

File: src/languages/javascript.js (hunk 3 above)
Comment (joshgoebel) [inline src/languages/javascript.js:364]:
```suggestion
        "super"
```

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@nathnolt Thanks so much, I went ahead and pushed this all the way home.

**Comment by nathnolt** [comment]
@joshgoebel thank you

**Pull request merged**